lib/Fmt_ast.ml

```diff
@@ -2736,7 +2740,9 @@ and fmt_class_expr c ?eol ?(box = true) ({ast= exp; _} as xexp) =
   @@
   match pcl_desc with
   | Pcl_constr (name, params) ->
-      let params = List.map params ~f:(fun x -> (x, Invariant)) in
+      let params =
+        List.map params ~f:(fun x -> (x, (NoVariance, NoInjectivity)))
```

Comment:
if we pass tuples as suggested in another comment, this tuple could be named default_variance_and_injectivity (or Variance_and_injectivity.default next to a Variance_and_injectivity.fmt). This could use a better name though. Just Variance maybe?

Reply:
Done